examples/content_types/src/tests.rs

```diff
@@ -0,0 +1,52 @@
+use super::rocket;
+use super::serde_json;
+use super::Person;
+use rocket::http::{ContentType, Method, Status};
+use rocket::testing::MockRequest;
+
+fn test(uri: &str,
```

Comment:
Can we shorten the variable names so the declaration fits in one line? `status` and `body` are fine.